### PR TITLE
Split up convert-to-vega-spec.ts

### PR DESCRIPTION
**Pull request by NickLanam**
Summary: This was a 2800 line file. Now several smaller ones.

Relevant Issues: N/A

Type of change: /kind cleanup

Test Plan: Smoke/Unit tests. Nothing actually changes here; just some
constants and functions moving to new files.

Signed-off-by: Nick Lanam <nlanam@pixielabs.ai>

### CODE REVIEW

File: src/ui/src/containers/live/vega-spec-converters/convert-to-vega-histogram.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
+/*
```
Comment (aimichelle) [inline src/ui/src/containers/live/vega-spec-converters/convert-to-vega-histogram.ts:1]: Maybe a little redundant to have these files called `convert-to-vega-*` when they're already nested inside the `vega-spec-converters` directory.
How about just `vega-spec-converters/histogram.ts`?

Comment (NickLanam) [reply src/ui/src/containers/live/vega-spec-converters/convert-to-vega-histogram.ts]: Done.

**Review by aimichelle — DISMISSED** [review]

**Comment by vihangm** [comment]
test this please

**Review by JamesMBartlett — APPROVED** [review]
I assume you tested that all these chart types still work

**Comment by NickLanam** [comment]
> I assume you tested that all these chart types still work

Yep, no differences found for any of them

**Review by aimichelle — APPROVED** [review]

**Pull request merged**